en/subscription_services_tos.md

```diff
@@ -40,7 +40,7 @@ You may cancel your subscription to the Services at any time. You can do this by
 We only offer Mozilla VPN as a paid subscription service.
 
 ### You Must Be Eligible
-You must be in a country where the Service is available to download Mozilla VPN and use the Service. Currently, the Service is available in Austria, Belgium, Bulgaria, Canada, Croatia, Cyprus, Czech Republic, Denmark, Estonia, Finland, France, Germany, Hungary, Ireland, Italy, Latvia, Lithuania, Luxembourg, Malaysia, Malta, the Netherlands, New Zealand, Poland, Portugal, Romania, Slovakia, Slovenia, Singapore, Spain, Sweden, Switzerland, the United Kingdom, and the United States.  
+You must be in a country where the Service is available to download Mozilla VPN and use the Service. Currently, the Service is available in Australia, Austria, Bangladesh, Belgium, Brazil, Bulgaria, Canada, Chile, Colombia, Croatia, Cyprus, Czech Republic, Denmark, Egypt, Estonia, Finland, France, Germany, Greece, Hungary, India, Indonesia, Ireland, Italy, Kenya, Latvia, Lithuania, Luxembourg, Malaysia, Malta, Mexico, Morocco, the Netherlands, New Zealand, Nigeria, Norway, Poland, Portugal, Romania, Saudi Arabia, Senegal, Slovakia, Slovenia, Singapore, South Africa, South Korea, Spain, Sweden, Switzerland, Thailand, Taiwan, Türkiye, Uganda, Ukraine, the United Kingdom, the United States, and Vietnam.
```

Comment:
@mafeld swap Thailand and Taiwan

Reply:
ok. I believe I have proposed that change. Thank you